io.dbeaver.DBeaverCommunity.Client.mariadb.json

```diff
@@ -11,7 +11,10 @@
       "name": "mysql",
       "buildsystem": "cmake",
       "build-options": {
-        "strip": true
+        "strip": true,
+        "build-args": [
+          "--share=network"
```

Comment:
build-args are not allowed. the build MUST be offline.

Reply:
You're right, my bad. I fixed it with a new commit, thank you for your feedback.